### ISSUE TITLE
Is there an easy way to have captions appear beneath the video instead of in an overlay?

### ISSUE THREAD

**Issue by blrichwine** [ableplayer/ableplayer#142]
Is there an easy way to have captions appear beneath the video instead of in an overlay? I don't see an option for that. I have video content that has text and other critical visual info at the bottom of the video which is being overlaid by the captions.

**Comment by jcoyne** [comment]
Here's another reason we need this. A vtt file that we don't control has many lines of text for a single cue. It blocks the whole video.  Ideally I'd be able to turn the overlay off an just have an external transcript viewer like in http://ableplayer.github.io/ableplayer/tests/test7b.html

<img width="484" alt="screen shot 2015-09-10 at 10 58 15 am" src="https://cloud.githubusercontent.com/assets/92044/9793413/ef5d2342-57aa-11e5-9e11-863f32b7db96.png">

**Comment by jcoyne** [comment]
Currently the overlay has `display: block` on the element itself. So I can't easily hide it by doing this (due to the greater precedence given to element styles):
```css
   .able-captions {
     display: none;
   }
```

I was able to put in a gross hack to hide it:
```css
  .able-captions {
    left: -1000px;
  }
```

**Comment by terrill** [comment]
I agree that there's a need for this and have tagged it as a high priority enhancement. We'll need to rework the structure of the player so the caption container is independent.

**Comment by blrichwine** [comment]
That would be great. If reworking the structure of the player, it would be good to make it responsive to changes in the page layout at the same time. Right now the code forces the player to stay a fixed size. We had to rewrite the rendering pieces to make the player responsive before our communications office would let us use it. 

Now that we have our implementation of ableplayer working in a responsive environment, I was going to separate out the caption rendering elements. Unfortunately ableplayer's code has changed considerably since we did our responsiveness work so it won't be possible to merge it. 

One can see our responsive player by playing with the browser window or launching on various mobile devices here:
https://atac.iu.edu/

By responsive we mean:
* The size of the player scales to the space allotted to it
* all elements of the player resize on the fly even while playing

Supporting responsive layouts might be a separate issue, but if wanting to support it, it would need to be kept in mind when separating out the captioning elements.
